packages/schemas/src/types/user.ts

```diff
@@ -18,9 +18,24 @@ export const userInfoSelectFields = Object.freeze([
   'isSuspended',
 ] as const);
 
-export const userInfoGuard = Users.guard.pick(
-  Object.fromEntries(userInfoSelectFields.map((key) => [key, true]))
-);
+/**
+ * The `pick` method of previous implementation will be overridden by `merge`/`extend` method, should explicitly specify keys in `pick` method.
```

Comment:
i doubt it's due to type losing. an `as` is actually better:

```ts
Object.fromEntries(userInfoSelectFields.map((key) => [key, true])) as Record<userInfoSelectFields[number], true>
```

Reply:
It works 💯